service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinNotificationEmailService.kt

```diff
@@ -0,0 +1,169 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.messaging.bulletin
+
+import fi.espoo.evaka.daycare.domain.Language
+import fi.espoo.evaka.emailclient.IEmailClient
+import fi.espoo.evaka.shared.async.AsyncJobRunner
+import fi.espoo.evaka.shared.async.SendUnreadBulletinNotificationEmail
+import fi.espoo.evaka.shared.db.Database
+import mu.KotlinLogging
+import org.jdbi.v3.core.kotlin.mapTo
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Service
+import java.time.Instant
+import java.util.UUID
+
+private val logger = KotlinLogging.logger { }
+
+@Service
+class BulletinNotificationEmailService(
+    private val asyncJobRunner: AsyncJobRunner,
+    private val emailClient: IEmailClient,
+    env: Environment
+) {
+    init {
+        asyncJobRunner.sendBulletinNotificationEmail = ::sendBulletinNotification
+    }
+
+    val fromAddress = env.getRequiredProperty("mail_reply_to_address")
```

Comment:
Not changed here so won't block but: it's better to use `fi_espoo_...` style names for eVaka specific configs to avoid any possible conflicts with other packages and to clarify which configs are meant for eVaka source code vs. other systems.

If you decided to change this, it will require a "migration" in evaka-infra: define both variations of the environment variable until only the new one is supported and deployed everywhere.

Reply:
Actually it seems that this env variable is defined only locally on the app level, ie. is not in evaka-infra 🤔

Reply:
Even better